packages/component-base/src/style-props.js

```diff
@@ -38,6 +39,9 @@ addGlobalThemeStyles(
 
         --vaadin-focus-ring-width: 2px;
         --vaadin-focus-ring-color: var(--_vaadin-color);
+
+        --_vaadin-icon-cross: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor"><path stroke-linecap="round" stroke-linejoin="round" d="M6 18 18 6M6 6l12 12" /></svg>');
+        --_vaadin-icon-warn: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor"><path stroke-linecap="round" stroke-linejoin="round" d="M12 9v3.75m-9.303 3.376c-.866 1.5.217 3.374 1.948 3.374h14.71c1.73 0 2.813-1.874 1.948-3.374L13.949 3.378c-.866-1.5-3.032-1.5-3.898 0L2.697 16.126ZM12 15.75h.007v.008H12v-.008Z" /></svg>');
```

Comment:
That is something completly new, or? I'm wondering if this is within the base-theme without a "simple" way to hide it / go back to the previous version without icon

Reply:
I was suprised by this :) because now it's an internal property and lumo would probably override an internal property to `none` to make it backward compatible. I'm not against this change, I kinda like it. I would just prefer a "public api" / official customizing of it. In this example it could be `--vaadin-input-field-error-icon: svg / none` to make it easy for people to customize it if needed. 

For example we have a similar use-case where we show different error-icons depending on the severity of an error (Flow's Binder's ErrorLevel) allowing us to show a big red cross or blue explanation mark as information.